iosApp/iosApp/Explore/RepoListViewModel.swift

```diff
@@ -7,21 +7,45 @@
 //
 
 import Foundation
-import Shared
 import ExploreShared
 
 @MainActor final class RepoListViewModel: ObservableObject {
     @Published var repoList: [Repo] = []
     
+    let pageSize: Int32 = 40
+    
+    var viewerRepo: ViewerRepository
+    var isLastPage: Bool = false
+    var lastCursor: String? = nil
+    
+    static func build(_ repos: [Repo]) -> RepoListViewModel {        
+        return RepoListViewModel(viewerRepo: FakeViewerRepository(repos: repos))
+    }
+    
+    static func buildReal() -> RepoListViewModel {
```

Comment:
I usually put the static / class functions at the end of the file, but it's your call here.